src/linker/Linker.Dataflow/FlowAnnotations.cs

```diff
@@ -386,6 +387,16 @@ TypeAnnotations BuildTypeAnnotations (TypeDefinition type)
 			return new TypeAnnotations (type, typeAnnotation, annotatedMethods.ToArray (), annotatedFields.ToArray (), typeGenericParameterAnnotations);
 		}
 
+		private IReadOnlyList<ICustomAttributeProvider>? TryGetGeneratedTypeAttributes (TypeDefinition typeDef)
```

Comment:
Nit: The way I know the rules if methods starts with "TryGet", it should return a bool and probably use out parameter to return the actual value. On the other hand "Get" method can return null if it makes sense as a missing value.

Reply:
Yeah, I admit that this is a personal thing -- I prefix with `Try` any time there's a potential for "no result." Happy to change it if that doesn't seem helpful to you, though.

Reply:
If the "no result" is an important bit I would go with TryGet returning a boolean - but again it's a subjective thing. That said - as is it would not pass FX API review ;-)